src/minoversikt/minoversikt-bruker-panel.tsx

```diff
@@ -154,19 +152,15 @@ function MinoversiktBrukerPanel({
                 </div>
             </div>
             <Collapse isOpened={apen}>
-                {erHuskelappFeatureTogglePa ? (
-                    <HuskelappPanel bruker={bruker} />
```

Comment:
Om du skal fjerne hele huskelappen fra no form for visning i panel må du også endre chevron på raden for bruker. Nå vil det være en chevron man kan klikke på, men som gir null handling

Reply:
Oida, den skulle ikkje forsvinne, eg skal prøve på nytt 😅

Reply:
Eg prøvar på nytt i ein ny PR: https://github.com/navikt/veilarbportefoljeflatefs/pull/988